CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
-### 0.3.4 (Next)
+### 0.4.1 (Next)
 
 * Your contribution here.
+
+### 0.4.0 (2018/07/27)
```

Comment:
This hasn't been released yet, so it shouldn't have a date and say `0.4.0 (Next)`.